integration-tests/sql/src/main/java/org/apache/camel/quarkus/component/sql/it/SqlRoutes.java

```diff
@@ -114,7 +123,9 @@ private Path createTmpFileFrom(String file) throws IOException {
             while ((c = is.read()) >= 0) {
                 baos.write(c);
             }
-            fos.write(baos.toByteArray());
+            String content = new String(baos.toByteArray());
```

Comment:
```suggestion
            String content = new String(baos.toByteArray(), StandardCharsets.UTF_8);
```

You never know what the current machine's default charset is.

Reply:
I'll fix it.

Reply:
done